tg-bot/src/main/java/com/tutrit/tgbot/example/repository/OrderRepository.java

```diff
@@ -0,0 +1,38 @@
+package com.tutrit.tgbot.example.repository;
+
+import com.tutrit.tgbot.example.bean.Order;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class OrderRepository {
+
+    private static final Logger logger = LoggerFactory.getLogger(OrderRepository.class);
+    private static final Map<Long, Order> orders = new HashMap();
+
+    public static Order createOrder(Order order){
+        orders.put(order.getId(), order);
+        return order;
+    }
```

Comment:
id should be increased depend on orders Map. 
when user saves new Order, it has't any id, value at that point is null.

I mean ids for new objects  should be generated on repository level.

Otherwise, new order should have id before it stored in repository, 
1. we should validate that id is not null
2. if Id is present - then decline creation and notify user that object cannot be created because of id duplication